.github/workflows/action_tests.yaml

```diff
@@ -145,3 +145,46 @@ jobs:
           cd local-action
           npm install
           ./action_tests/assert.js all-hold-the-line-no-upload-id
+
+  payload:
+    name: ${{ matrix.description }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - description: pull-request-payload
+            payload_path: action_tests/pull_request_test_payload.json
+          - description: trunk-merge-payload
+            payload_path: action_tests/trunk_merge_test_payload.json
+          - description: all-payload
+            payload_path: action_tests/all_test_payload.json
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: local-action
+
+      - name: Craft TEST_GITHUB_EVENT_PATH
+        shell: bash
+        run: |
+          echo "TEST_GITHUB_EVENT_PATH=local-action/${{ matrix.payload_path }}" >>$GITHUB_ENV
```

Comment:
let's try to craft `TEST_GITHUB_EVENT_PATH` using `payload_path` now using htis trick:

```
jq --null-input --argfile foo foo.json '{"inputs": ($foo | tostring)}'
```